### PR TITLE
DROOLS-2848 : Make the Verifier Web Worker connector reusable for different V&V types

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2848

Some changes I found necessary when adding the V&V for DMN tables. I removed the changes for the table. Just pushing these in so the code does not get stale.

The PR bundle:
https://github.com/kiegroup/kie-wb-common/pull/2295
https://github.com/kiegroup/drools-wb/pull/1004

### CODE REVIEW

**Comment by Rikkola** [comment]
Failing test: 

org.drools.workbench.screens.guided.dtable.client.widget.analysis.VerifierWebWorkerConnectionImplTest.firstActivationStartWebWorker

Taking a look.

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/DecisionTableAnalyzerBuilder.java [hunk 1]
```diff
@@ -74,11 +77,13 @@ public DecisionTableAnalyzer build() {
...
-                                                                           model,
-                                                                           new ModelMetaDataEnhancer(model).getHeaderMetaData(),
-                                                                           fieldTypeProducer.getFactTypes(),
-                                                                           ApplicationPreferences.getDroolsDateFormat()),
+            webWorker = new VerifierWebWorkerConnectionImpl(new DrlInitialize(UUID.uuid(),
+                                                                              model,
+                                                                              new ModelMetaDataEnhancer(model).getHeaderMetaData(),
+                                                                              fieldTypeProducer.getFactTypes(),
+                                                                              ApplicationPreferences.getDroolsDateFormat()),
+
```
Comment (manstis) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/DecisionTableAnalyzerBuilder.java]: Picky, but do we need the blank line?

Comment (Rikkola) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/DecisionTableAnalyzerBuilder.java]: No it should be removed.

**Comment by manstis** [comment]
@Rikkola Looks like it got stale quicker than you had hoped!?

Please fix conflicts and then ping me to test :-)

**Comment by Rikkola** [comment]
Bitten by myself I think. Will do.

**Comment by Rikkola** [comment]
@manstis updated, let's see if the tests pass.

**Comment by manstis** [comment]
@Rikkola This PR has "enforcer" issues (related to your changes) for "no-duplicate-declared-dependencies"

File: drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/pom.xml [hunk 2]
```diff
@@ -40,6 +40,11 @@
             <artifactId>drools-verifier-core</artifactId>
         </dependency>
 
+        <dependency>
```
Comment (manstis) [inline drools-wb-services/drools-wb-verifier/drools-wb-verifier-guided-decision-table-adapter/pom.xml]: This caused the enforcer problem.

**Comment by Rikkola** [comment]
jenkins please retest this

**Pull request merged**